amigos/amigos/cr100x_test.py

```diff
@@ -0,0 +1,5 @@
+import datetime
+from pycampbellcr1000 import CR1000
+device = CR1000.from_url('tcp:192.168.1.30:6785')
+data = device.get_data('Public')
```

Comment:
Well that was easy!

Reply:
yep. the data come as a dict. Thus formating is the only thing to do here